front/app/components/ProjectAndFolderCards/index.tsx

```diff
@@ -30,10 +29,14 @@ const ProjectAndFolderCards = ({
   showSearch = false,
   ...otherProps
 }: Props) => {
+  // used locally to keep track of the depth of the search
+  const [search, setSearch] = useState<string | null>(null);
```

Comment:
Moved this up so it can be passed in to the status count hook below, it's only used locally to store the search term and compute `rootLevelOnly`